### PR TITLE
Global settings

### PR DESCRIPTION
**Pull request by cljoly**
- feat: Add global settings for the commands
- doc: document the new global configuration
- style: run stylua

See the additional documentation in the Readme for usage instruction of this new feature.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +106,50 @@ use {
 
 ## Usage
 
-### list
+### Global Configuration
```
Comment (cljoly) [inline README.md:109]: @Chetan496 feel free to point out anything unclear in that section or anything that does not work for you.

Comment (Chetan496) [reply README.md:109]: > @Chetan496 feel free to point out anything unclear in that section or anything that does not work for you. (this is re: #38)

Thanks for this, this is very helpful.
I will try this feature soon.

**Comment by Chetan496** [comment]
Thanks a lot for this feature, will try it soon

**Comment by cljoly** [comment]
Great! I’m happy to refine this feature based on your input, feel free to share your thoughts on this.

**Comment by Chetan496** [comment]
It would be great to have this merged into master/main branch

**Comment by cljoly** [comment]
Sorry, I forgot that PR

**Pull request merged**